symfony/plugins/orangehrmCorePlugin/lib/form/widget/ohrmWidgetEmployeeNameAutoFill.php

```diff
@@ -149,94 +150,105 @@ public function render($name, $value = null, $attributes = array(), $errors = ar
-                        $this->generateId($name),
-                        $this->getEmployeeListAsJson($this->getEmployeeList()),
-                        $this->getHtmlId($name),
-                        $hiddenFieldId,
-                        $typeHint,
-                        $this->getOption('loadingMethod'),
-                        __('Loading'),                
-                        $this->generateId($name),
-                        url_for('pim/getEmployeeListAjax'),
-                        $ajaxRequiredPermissions);
-                        
-        
+            ,
+            $this->generateId($name),
+            $this->getEmployeeListAsJson($this->getEmployeeList()),
+            $this->getHtmlId($name),
+            $hiddenFieldId,
+            $typeHint,
+            $this->getOption('loadingMethod'),
+            __('Loading'),
+            $this->generateId($name),
+            url_for('pim/getEmployeeListAjax'),
+            $ajaxRequiredPermissions);
+
 
         return "\n\n" . $html . "\n\n" . $this->getHiddenFieldHtml($name, $empIdValue) . "\n\n" . $javaScript . "\n\n";
-        
+
     }
-    
-    protected function getHiddenFieldHtml($name, $value) {
-        
+
+    protected function getHiddenFieldHtml($name, $value)
+    {
+
         //$hiddenName = substr($name, 0, strlen($name) - 1) . '_id]';
         $hiddenName = $name . '[empId]';
-        $hiddenId   = $this->getHiddenFieldId($name);
-        
+        $hiddenId = $this->getHiddenFieldId($name);
+
         return "<input type='hidden' name='$hiddenName' id='$hiddenId' value='" . self::escapeOnce($value) . "' />";
-        
+
     }
-    
-    protected function getHiddenFieldId($name) {
-        
+
+    protected function getHiddenFieldId($name)
+    {
+
         return $this->generateId($name) . '_empId';
-        
+
     }
 
-    protected function getHtmlId($name) {
-        
+    protected function getHtmlId($name)
+    {
+
         if (isset($this->attributes['id'])) {
             return $this->attributes['id'];
         }
-        
+
         return $this->generateId($name) . '_empName';
-        
+
     }
-    
-    protected function getEmployeeList() {
-        
+
+    protected function getEmployeeList()
+    {
+
         $employeeList = $this->getOption('employeeList');
         $loadingMethod = $this->getOption('loadingMethod');
         $requiredPermissions = $this->getOption('requiredPermissions');
-        
+
         if (is_array($employeeList) || $employeeList instanceof Doctrine_Collection) {
             return $employeeList;
         }
 
-        if( $loadingMethod != 'ajax'){
-            $properties = array("empNumber","firstName", "middleName", "lastName", "termination_id");
-            $employeeList = UserRoleManagerFactory::getUserRoleManager()->getAccessibleEntityProperties('Employee', 
-                    $properties, null, null, array(), array(), $requiredPermissions);
-        
+        if ($loadingMethod != 'ajax') {
```

Comment:
I just added these lines. I do not know why GIT shows all these lines as changed.